build/python.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Functions.PythonWorkerRunEnvironments" Version="1.0.20191003.2" />
```

Comment:
Microsoft.Azure.Functions.PythonWorkerRunEnvironments - is this the name of nuget package? why not just Microsoft.Azure.Functions.PythonWorker?

Reply:
Version does not look right. Should it be: 1.0.20191003?

Reply:
> Microsoft.Azure.Functions.PythonWorkerRunEnvironments - is this the name of nuget package? why not just Microsoft.Azure.Functions.PythonWorker?

This was the previous name when we were packaging directly with the core tools. I'll change it to Microsoft.Azure.Functions.PythonWorker.

Reply:
> Version does not look right. Should it be: 1.0.20191003?

The version is correct. The `.2` at the end is meant to account for the build number for that particular day. For example, if you do multiple builds in a day, the `.x` keeps the build number unique.

Reply:
We should use valid Semantic versioning as described here: https://semver.org. Need to fix the build to produce a valid version

Reply:
> We should use valid Semantic versioning as described here: https://semver.org. Need to fix the build to produce a valid version

Will modify it to remove the `.` after the date so that it still remains unique but follows correct patch version syntax.